pkg/virt-handler/cgroup/cgroup_test.go

```diff
@@ -72,6 +75,11 @@ var _ = Describe("cgroup manager", func() {
 	BeforeEach(func() {
 		ctrl = gomock.NewController(GinkgoT())
 		rulesDefined = make([]*devices.Rule, 0)
+		v2DirPath = "/sys/fs/cgroup/"
+	})
+
+	AfterEach(func() {
+		v2DirPath = ""
```

Comment:
Can we define `v2DirPath` as a const instead?

Reply:
I am tweaking it in the test

Reply:
Ah, missed it :man_facepalming: 
Thanks!